libs/solid/skeleton/src/lib/components/landing/landing.component.ts

```diff
@@ -73,4 +80,21 @@ export class LandingComponent {
       return this.limitedMessages;
     });
   }
+
+  public ngAfterViewInit(): void {
+    setTimeout(() => {
+      this.introService.guidedTour((_targetElement: any) => {
+        const landing = this.Landing?.nativeElement;
+        const menuOffSetTop =
+          document.getElementById('menu-grid-list')?.offsetTop;
+        if (_targetElement.id == 'menu-tile-0') {
+          if (menuOffSetTop) landing.scrollTop = menuOffSetTop - 50;
```

Comment:
Ist das der Scroll-"Hack"?

Reply:
Works for me - aber wir sollten das Problem trotzdem an introJS melden.

Reply:
Wenn du Zeit dafür hast 👍 Ich habe das auch schon notiert, damit es nicht vergessen geht